plugins/playlist/src/components/PlaylistPage/PlaylistEntitiesTable.tsx

```diff
@@ -73,16 +74,22 @@ export const PlaylistEntitiesTable = ({
   const removeEntity = useCallback(
     async (_, entity: Entity | Entity[]) => {
       try {
+        const entityName = [entity].flat().map(item => item.metadata.name);
```

Comment:
It's preferable to show the UI friendly title for entities in display messages if it is defined
```suggestion
        const entityName = [entity].flat().map(item => item.metadata.title ?? item.metadata.name);
```